### PR TITLE
fix: removed close button when handleCancel is not set

### PR DESCRIPTION
**Pull request by danlevings**
Beforehand we would always include a close button.
Even when there was no function for this close button.
Instead, remove the close button if the user has not
provided the prop 'handleCancel'

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-446.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-446.gearboxjs.preview.cogniteapp.com

File: src/components/SVGViewer/SVGViewer.tsx [hunk 1]
```diff
@@ -153,7 +154,7 @@ export class SVGViewer extends React.Component<SvgViewerProps, SvgViewerState> {
           {(isDesktop || !this.state.isSearchVisible) && (
             <StyledHeaderContainer>
               <MobileModalClose
-                data-test-id="close-svgviewer-btn"
+                data-test-id="close-svgviewer-btn-mobile"
```
Comment (KatyHouse) [inline src/components/SVGViewer/SVGViewer.tsx]: Is it necessary to identify if it's a desktop or mobile button?

**Comment by polomani** [comment]
@danlevings please, fix the commit name and I'll approve it
https://github.com/cognitedata/gearbox.js/blob/master/CONTRIBUTION.md

**Comment by valerii-cognite** [comment]
I agree with Katy's comment, this change will break our e2e-tests

**Comment by polomani** [comment]
@valerii-cognite why do you test gearbox outside of gearbox, shouldn't your tests not be dependent on component internal structure?

**Comment by valerii-cognite** [comment]
This is mostly due to gearbox lacking e2e testing functionality and us wanting to apply them. Any suggestions on how we can select appropriate item without relying onto implementation details are very welcome

**Comment by danlevings** [comment]
@polomani Can you give direction on how to rename commit messages?
Or I can make sure to do it write on merge

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-446.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-446.gearboxjs.preview.cogniteapp.com

**Comment by polomani** [comment]
@danlevings sorry, noticed too late. I guess you figured it out?

**Pull request merged**